common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -355,6 +356,13 @@
     String NO_EMAIL_RECIPIENTS =
         "Email notification is not sent as the number of recipients is zero.";
     String PARAMETER_MISMATCH = "Mismatch of given parameters: {0}.";
+    String ERROR_CONFIG_LOAD_EMPTY_STRING =
+        "Load configuration failed as empty string is passed as parameter.";
```

Comment:
Loading {0} configuration failed as empty string is passed as parameter.

e.g. {0} = user